libminifi/src/core/ProcessSession.cpp

```diff
@@ -626,6 +626,92 @@ bool keepSource,
   }
 }
 
+void ProcessSession::import(std::string source, std::shared_ptr<std::vector<std::shared_ptr<FlowFileRecord>>> flows,
```

Comment:
Since the vector itself is pretty light weight the caller can put it on the stack and pass it in by reference to avoid the surrounding shared pointer.